rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/parametric/InterpretedLanguageContributions.java

```diff
@@ -278,13 +284,26 @@ private static ISet loadContributions(Evaluator eval, LanguageParameter lang) {
         return (ISet) eval.eval(eval.getMonitor(), lang.getMainFunction() + "()", URIUtil.rootLocation("lsp"))
             .getValue();
     }
+ 
+    @Override
+    public CompletableFuture<IList> parseCommands(String command) {
```

Comment:
I like this duplication because it avoids java generics. better have it clear which non-terminal is used. Also in the new code a `list[CodeAction]` is used so it makes the functions more different.